test/regression/cases/quality_gate_idle/experiment.yaml

```diff
@@ -9,7 +9,7 @@ erratic: false
 target:
   name: datadog-agent
   cpu_allotment: 4
-  memory_allotment: 500MiB
+  memory_allotment: 250 MiB
```

Comment:
I think this is too low - `quality_gate_idle` nightlies are currently using around 350MiB.

Reply:
@GeorgeHahn specifically with reference to https://app.datadoghq.com/dashboard/vz3-jd5-bdi?fromUser=false&refresh_mode=paused&tpl_var_experiment%5B0%5D=quality_gate_idle&tpl_var_job_id%5B0%5D=64ae1a67-3095-4d08-9fb1-2269cb13a245&tpl_var_run-id%5B0%5D=64ae1a67-3095-4d08-9fb1-2269cb13a245&view=spans&from_ts=1737529601000&to_ts=1737530201000&live=false&tile_focus=6763738431426144.

Reply:
Oh, great catch. Yep, this LGTM.